### DISCUSSION TITLE
Extruder Not Working (Sometimes)

### DISCUSSION THREAD

**Discussion by JoeMaxey** [Jyers/Marlin#1742] [General Discussion]
This may very well be caused by something else, but wanted to see if this might be related to Jyers in any way. I'm running an E3V2 4.2.2 with a MS DD and CR Touch with v2.01 of Jyers. Many times now I have started a print, only to have the extruder motor just not engaging. Many prints work just find (although having a ton of trouble with this Overture PETG), but occasionally the print will start and the extruder motor just isn't turning at all. Not that the filament is clogged, the extruder motor doesn't turn. Have ran the same gcode with success on most prints and then fails (no extruder turning/engaged) on others. Any guesses what I may have done wrong?

UPDATE, just started a print and got the waste line as well as the skirt printed just fine. As soon as it tries to print the object the extruder stopped turning again. Seems so odd to me that the extruder would work  just fine for the waste/skirt, only to stop when it comes to the object.

**Comment by Sonicboom247** [comment]
same thing just happened to me, E3 V2 4.2.2 was fine yesterday when i was using my custom MBL no probe firmware, then added a BLtouch, got it working on my edited Jyers awesome firmware but was not able to get UBL to probe correctly.  i then downloaded E3V2-UBL-BLTouch-10x10-HS-v4.2.2-v2.0.1, BLTouch working just fine but when i start a print, zero movement from Extruder.  However i can heat up the nozzle and extrude manually.
